### ISSUE TITLE
inpost 

### ISSUE THREAD

**Issue by john-peterson** [john-peterson/john-peterson#7]
This has absolutely nothing to do with computers

Well everything has but this is COMPLETELY out of my hands

And this must be the first time I talk about something non computing here

I don't know I am a PAYING customer so how hard is it to put up with my simple LOWERCASE message

**Comment by john-peterson** [comment]
My fiber ⛺ poles broke as they always do

I finally after five years realised that I have to remove the rubber cord and deal with more assembly time and the potential DISASTER that a tent pole is lost

Each strand has seven pieces. Count your blessings and pieces every time

So to the point

Inpost

They return

I was six hours late and they returned it

They are hard core set on keeping purging them after two days

It's like Wilford Bromley in Seinfeld

They take their job seriously. Apes.... And now I have to be careful that not the wrong person read this. People like unnamed world leaders have a 🥜 brain compared to me. I'm marginally smarter than other unnamed billionaires. 

And scientists of course..... .....

....

....

So

**Comment by john-peterson** [comment]
It's unbelievable that I have not been able to add my email to the app so I can ask for a sane storage period

I can't run RUN back and forth like the best apes in the trees 

I walk slowly like

**Comment by john-peterson** [comment]
And just like everyone EVERYTHING this is a computer problem is it not. Can you claim this is off topic. Just because I have not posted any code. Contributed any coding for this

The app is a disaster

The LOCKER DOESN'T EVEN SAY THE PARCEL WAS RETURNED TEN MINUTES AGO because it's now almost twenty one minutes past your deadline and almost the entire attention span of the smartest ape in this country since you ordered placed the order (59 minutes ago)

**Comment by john-peterson** [comment]
So I'm

It's getting dark and I'm in front of the locker to retrieve survival items. And

Error WRONG CODE


That's the message after they removed it

Anyone here worked for ups? Anyway (just remembered I read something from an actual worker in THEIR LANGUAGE) need to find a soundboard for a delivery driver to understand this

Anyway

So the locker message is wrong number

That's for the we code

And everything is so sleak how could they complain about so much polish and miss the 

Feel like I'm preaching like Jesus here before he killed himself. Did Jesus actually commit suicide

He just needed to get away JUST WALK IT OFF

anyway

**Comment by john-peterson** [comment]
This is why my Reddit eventually got deleted I think

I'm not in control over it

World champion genes. 

I actually don't feel like stop talking

**Comment by john-peterson** [comment]
So what is the point here

It's something I could patch. Theoretically

And I can't get away from this problem

Believe me and you can be more confident about the truth value in this statement than anything in this universe

I have spent all my energy and resources and attention and cunning to GET AWAY FROM THE OTHER mammals of my species   that simply don't. Lack . don't have it. Lack consciousness. They are not burdened by consciousness arousal wakefulness awareness attention consciousness knowledge. Awareness THEY LACK AWARENESS to be able to eat and live and procreate

**Comment by john-peterson** [comment]
Believe me death is a relief for a conscious creature on this planet and most likely all or any planet. Any planetary being

**Comment by john-peterson** [comment]
When there is an invasive species that create. Are enormously excitable and LOUD

LOUD

LOUD

LOUD

EVERYWHERE

UNTIL DEATH BECOMES THE ONLY OPTION

**Comment by john-peterson** [comment]
TLDR

I have a problem with quieting down when the other apes get angry. Tolerance is the key word. I'm not in control of long messages like this. Off topic

The locker is bugged   it should say your parcel was returned. It says wrong key. When the key is deleted   I guess it makes sense. If someone delete a file mv file is wrong. The error message is SUPPOSED TO BE MISLEADING no one has ever the whole point of having the power to write an error message is to great to not to NOT not create a misleading message

The way they manage capacity is incomprehensible

There are lockers in the forest that has ten delivers per billion years. Two days returns

They have lockers next to a two thousand meter tall house with ten million deliveries per second ALSO TWO DAYS RETURNS the system can not support any differencees

NO DIFFERENCES CAN BE TOLERATED

**Comment by john-peterson** [comment]
And the why stop here

There is so much potential

For intelligence awareness

I would read with interest a complete explanation of the reasons behind slow response times on public computers like the locker computer screen

Every single one I can remember have either a touch code from.. well... Just incredible bad. Or. Like that I want to know what people do. Qt is incredibly boring for me

**Comment by john-peterson** [comment]
And it switched back to English by action

The reason might simply be. Whatever

I just can't stop analysing dumb animals. I think it's my upbringing to blame that humans are not dumb animals   Somehow I can't unlearn. Keep racking torturing my brain over an everyday tool that is good enough for the local populace and fit the AVERAGE

**Comment by john-peterson** [comment]
So of course it has to have a timeout for changing back to the local language

I'm new here

If I switch to English it means I have no idea what's going on here I. It's like stumbling into  Oświęcim   I need a briefing

So while. I am still operating the machine back and forth. A one minute timeout would be fine. But every return to the main menu changed back the language

**Comment by john-peterson** [comment]
Here you get your Christmas eve order on Christmas day morning. And it's returned on boxing Day if you were slow to pick it up. Or out of town. Had to travel away from home. Or possibly other reason unusual reasons

It's better than  Oświęcim

**Comment by john-peterson** [comment]
And Jesus could not create the United Nations

It was created in San Francisco several thousand years later

**Comment by john-peterson** [comment]
Animals running around here now in the dark

They are scared out of their minds

They are very good at hiding in the long daylight

Or rather. That's what they do. They are neither good nor bad at it per se. They just. They are there   they exist.  Because of reasons

Stumbling in the dark and I'm annoyed because it did not smell me earlier and walked so close to me
